specification.md

```diff
@@ -1251,7 +1253,7 @@ Let's take a look at an example of such definitions:
     "type": "expression"
   },
   {
-    "name": "isMinor",
+    "name": " is a minor ?",
```

Comment:
space between minor and ?

Reply:
The expression function `name` property type is string, so the name of the expression function can be any arbitrary string including spaces, dots, colons, and question marks. It can end with a space and it can start with a space. When it does, it should be different from when it doesn't, e.g. `"aFunction"` is different from `" aFunction"`